src/runtime/entries/firebase-v2.ts

```diff
@@ -0,0 +1,10 @@
+import "#internal/nitro/virtual/polyfill";
+import functions from "firebase-functions/v2";
+import { toNodeListener } from "h3";
+import { nitroApp } from "../app";
+import { useRuntimeConfig } from "#internal/nitro";
+
+export const server = functions.https.onRequest(
+  useRuntimeConfig()?.firebase?.httpRequestOptions || {},
```

Comment:
I believe this line needs an update, now that the `firebase` config is no longer under the `runtimeConfig` Nitro setting. Because as I am testing it right now, my config of a different-than-default region is not being respected in the emulator.

Reply:
@sisou do you know how to access the nitro config from this entry file? I can't work it out... @Hebilicious any advice?

Reply:
I guess the nitro config is build time whereas we're trying to target runtime here... So maybe it should be in runtimeConfig?

Reply:
You're right I missed that 🤦🏽 I don't think we can access the Nitro options here.

In the preset file, you can do something similar to vercel with the `rollup:before` hook : 
And you should be able to use the rollup replace plugin to inject the configuration...

If this doesn't work, I have a few other suggestions : 

1) Use the rollup config, add these options to the virtual files, and import them in the entry file. 
2) Leave the options in the Nitro config, attach them to the AppConfig somewhere in the build, then import that in your preset
3) Use the runtimeConfig key directly.

But I think @pi0 advice would be very valuable here, as I'm not sure what's the best way. Probably 2).